### PR TITLE
add flux-lptest

### PR DESCRIPTION
**Pull request by garlick**
Problem the `lptest` utility is useful for testing standard I/O buffering on a deployed system, but it only exists in the source tree.

At least, I keep finding a use for it and accessing it is not always convenient.

This elevates the `lptest` test program to a flux command.

I didn't feel a man page was warranted for this one.

### CODE REVIEW

**Review by grondo — APPROVED** [review]
Sure, why not? :+1:

File: src/cmd/builtin/lptest.c [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        cmd_lptest,
+        "LINELENGTH LINECOUNT",
+        "Create lines of regular output for standard I/O testing",
+        0,
+        NULL);
+    return (e == OPTPARSE_SUCCESS ? 0 : -1);
+}
+
+// vi:ts=4 sw=4 expandtab
+
```
Comment (grondo) [inline src/cmd/builtin/lptest.c:73]: Python linter maybe didn't like this trailing line? :shrug:

Comment (garlick) [reply src/cmd/builtin/lptest.c:73]: What it's checking C files now??? Ok.

**Comment by garlick** [comment]
OK well that did it.   I'll set MWP - thanks!

File: src/cmd/builtin/lptest.c (hunk 1 above)
Comment (grondo) [reply src/cmd/builtin/lptest.c]: Oh, I guess this set of [pre-commit](https://pre-commit.com/) hooks are not Python specific:
```yaml
    hooks:
      - id: check-added-large-files
      - id: check-case-conflict
      - id: check-docstring-first
      - id: check-shebang-scripts-are-executable
      - id: end-of-file-fixer
      - id: trailing-whitespace
      - id: mixed-line-ending
```

Comment (garlick) [reply src/cmd/builtin/lptest.c]: I did not realize that!

**Pull request merged**